playbooks/roles/cluster/defaults/main.yml

```diff
@@ -63,16 +63,19 @@ cluster_job_template_post_install_type: 'run'
 cluster_job_template_post_install_playbook: 'playbooks/cluster_create_post_install.yml'
 cluster_job_template_post_install_project: '{{ tower_configuration_project_name }}'
 
-cluster_workflow_job_template_name: "Cluster_provision_preflight"
+cluster_workflow_credential_job_template_name: "Cluster_provision_bootstrap"
+cluster_workflow_credential_job_template_desc: "Configure cluster provision workflow for a given cluster name"
+cluster_workflow_credential_job_template_playbook: "playbooks/cluster_provision_bootstrap.yml"
+
+cluster_workflow_job_template_name: "Cluster_provision"
+cluster_workflow_name: "Provision_workflow"
```

Comment:
Just running through another cluster create job, in terms of naming: thinking it might be clearer to set `cluster_workflow_name ` as `Cluster_provision_{cluster-name}` and `cluster_workflow_job_template_name` as `Cluster_provision_preflight_{cluster-name}`

At the moment the generated workflow name is Provision_workflow_{cluster-name} and the first job in the workflow is called `Cluster_provision_{cluster-name}` which is slightly confusing